centinel/primitives/dnslib.py

```diff
@@ -15,7 +15,7 @@ def get_ips(host, nameserver=None, record="A"):
     return lookup_domain(host, nameservers=nameservers, rtype=record)
 
 
-def lookup_domain(domain, nameservers=[], rtype="A", timeout=10):
+def lookup_domain(domain, nameservers=[], rtype="A", timeout=2):
```

Comment:
> We already weren't going to have one clean pcap

We are, to some extent. The main usecase for this is to run on dedicated hardware periodically.

> if you store a lot of data, then larger files are better

Not true. We are doing a `sync` once every hour. Smaller files are transferred more reliably that way.

> I don't think we are talking about a large enough pcap for upload problems to be significant either.

Also not true as I ran into this issue while running the benchmark on test nodes in the field.